src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,257 @@
-  - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
-freshnessValidatedDate: never
+    - /docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
+    - /docs/integrations/kubernetes-integration/understand-use-data/understand-use-data
+    - /docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#services
+freshnessValidatedDate: 2024-06-24
 ---
 
-## Reduce data ingest [#reducedataingest]
+If you've checked your data ingestion and you'd like to reduce the amount you have, you can do it through charts and through parameters.
 
-Our charts support setting an option to reduce the amount of data ingested at the cost of granularity. To enable it, set `global.lowDataMode` to `true` in the `nri-bundle` chart.
+You can reduce the data in your charts and works for all installation methods: the guided install, Helm, or Manifest. Our charts can help you adjust settings to reduce the data you take in, which may result in less detailed information. These settings allow you to focus on the most important data, which can help you identify the most important insights.
 
-`lowDataMode` affects four specific components of the `nri-bundle` chart outlined below.
+It's also possible to reduce data ingestion through parameters. However, the modification of the parameters will depend on the method used to install Kubernetes.
 
-### New Relic Infrastructure
+## Reduce data ingested by setting `global.lowDataMode` [#reduce-data-ingest]
 
-If `lowDataMode` is enabled, the default scrape interval changes from `15s` to `30s`.
-You can also specify a custom value for it using `config.interval`, which will take preference over `lowDataMode`.
+If you're looking to cut costs, you can modify the `global.lowDataMode` key value. This will cut down on the number of metrics sent. The modification of this key will depend on the method used to install Kubernetes:
 
-### Prometheus Agent Integration
+    * **Guided install** (New Relic CLI, Helm, Manifest). The`global.lowDataMode` key is enabled by default.
 
-If `lowDataMode` is enabled, the metrics that are prefixed with the following are excluded by default as they're already collected and used by the [New Relic Kubernetes Integration](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data/#event-types).
+    * **Helm**. Set the `global.lowDataMode` key in the [`nri-bundle` chart](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) to true.
 
+    * **Manifest**. Set the `global.lowDataMode` key in the Manifest configuration yaml file to true. If you followed our [installation instructions](/install/kubernetes/?dropdown1=manifest), the file was called `newrelic-manifest.yaml`.
+
+`lowDataMode` key affects these specific components of the `nri-bundle` chart:
```

Comment:
Can we include a link to the nri-bundle Helm Chart?
https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle

Reply:
Sure! Added.